packages/website/blog/2024-11-30-catching-any.mdx

```diff
@@ -0,0 +1,237 @@
+}
+
+rejectWith('Nandor').catch(error => {
+  //                       ~~~~~
+  // eslint(@typescript-eslint/use-unknown-in-catch-callback-variable):
+  // Prefer the safe `: unknown` for a `catch` callback variable.
+
+  console.log(error.message); // Should be a type error, but isn't
+});
+```
+
+Had the `error` been given a `: unknown` type, TypeScript would be able to report on `error.message` as being unsafe.
+
+## Banning Usage of Unsafe Types
+
+Once an `any` type exists in code, it is "infectious": it can turn the types of values based on it into more `any`s.
+typescript-eslint includes a collection of rules that flag code patterns which spread or are generally made less safe due to an `any`.
+
+### Banning Unsafe Arguments
+
+[`@typescript-eslint/no-unsafe-argument`](https://typescript-eslint.io/rules/no-unsafe-argument) reports on passing an `any` typed value to a function call.
+
+For example, given some value returned as `any` by `JSON.parse`, the rule would report on it being passed to a function with a more precise parameter type:
+
+```ts
+declare function greet(friend: string): void;
+
+function greetFromData(raw: string) {
+  greet(JSON.parse(raw));
+  //    ~~~~~~~~~~~~~~~
+  // eslint(@typescript-eslint/no-unsafe-argument):
+  // Unsafe argument of type `any` assigned to a parameter of type `string`.
+}
+```
+
+### Banning Unsafe Assignments
+
+[`@typescript-eslint/no-unsafe-assignment`](https://typescript-eslint.io/rules/no-unsafe-assignment) reports on assigning an `any` typed value to a property or variable.
+
+For example, given some value returned as `any` by `JSON.parse`, the rule would report on it being assigned to an object's property:
+
+```ts
+const person = {
+  name: '(anonymous)',
+};
+
+function setNameFromData(raw: string) {
+  person.name = JSON.parse(raw);
+  // ~~~~~~~~~~~~~~~~~~~~~~~~~~
+  // eslint(@typescript-eslint/no-unsafe-argument):
+  // Unsafe assignment of an `any` value.
+}
+```
+
+### Banning Unsafe Calls
+
+[`@typescript-eslint/no-unsafe-call`](https://typescript-eslint.io/rules/no-unsafe-call) reports on assigning an `any` typed value to a property or variable.
+
+For example, given some value returned as `any` by `JSON.parse`, the rule would report on it being assigned to an object's property:
+
+```ts
+const person = {
+  name: '(anonymous)',
+};
+
+function setNameFromData(raw: string) {
+  person.name = JSON.parse(raw);
+  // ~~~~~~~~~~~~~~~~~~~~~~~~~~
+  // eslint(@typescript-eslint/no-unsafe-argument):
+  // Unsafe assignment of an `any` value.
+}
+```
+
+### Banning Unsafe Member Accesses
+
+[`@typescript-eslint/no-unsafe-member-access`](https://typescript-eslint.io/rules/no-unsafe-member-access)
+
+### Banning Unsafe Returns
+
+[`@typescript-eslint/no-unsafe-return`](https://typescript-eslint.io/rules/no-unsafe-return)
+
+Each of those rules target a different unsafe practice around the `any` type
+
+## Additional Helpers
+
+### Disabling TypeScript Suppressions
```

Comment:
I'm not 100% sure I want these additional `any` area points in this blog post. I _think_ it makes sense to include them, but it does add to the length of the post... Unsure but leaning towards inclusion. 🤷